skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,349 @@
+# Module for the BNL image processing project
+# Developed at the NSLS-II, Brookhaven National Laboratory
+# Developed by Gabriel Iltis, Oct. 2013
+"""
+This module is designed to facilitate image arithmetic and logical operations
+on image data sets.
+"""
+
+import numpy as np
+import parser
+

```

Comment:
might as well import the local_* family from numpy

```python
from np import logical_and, logical_or, logical_not, logical_xor
```
and define an `__all__` that lives at the end of this module and includes all logical_* operations
```python
__all__ = [logical_and, logical_or, logical_not, logical_xor, logical_nand, logical_nor, logical_sub]
```